src/tools/pub/cmd/pub-get.md

```diff
@@ -143,6 +143,17 @@ command-line argument, as mentioned above.
 For options that apply to all pub commands, see
 [Global options](/tools/pub/cmd#global-options).
 
+### `--dry-run` or `-n`
+
+With this, pub reports the dependencies that would
+but does not make the changes. This is useful if you
```

Comment:
Let's start these with a _-s_ verb. Also, we use [contractions](https://developers.google.com/style/contractions) (especially for _not_) and [semantic line breaks](https://github.com/dart-lang/site-shared/blob/master/doc/writing-for-dart-and-flutter-websites.md#semantic-line-breaks).

```suggestion
Reports the dependencies that would be made,
but doesn't make the changes.
This option is useful if you
```